terminal.go

```diff
@@ -526,31 +526,57 @@ func (g *LightningTerminal) start() error {
 		return fmt.Errorf("error displaying startup info: %v", err)
 	}
 
-	// Wait for lnd to be unlocked, then start all clients.
-	select {
-	case <-readyChan:
+	// waitForSignal is a helper closure that can be used to wait on the
+	// given channel for a signal while also being responsive to an error
+	// from the error Queue, LND quiting or the interceptor receiving a
+	// shutdown signal.
+	waitForSignal := func(c chan struct{}) error {
+		select {
+		case <-c:
```

Comment:
nit: return `nil` here (and remove the `return` at the end of the closure) to make things even more explicit?

Reply:
good idea :) done!